src/VisualStudio/CSharp/Impl/Snippets/CSharpSnippetInfoService.cs

```diff
@@ -31,7 +31,7 @@ internal class CSharpSnippetInfoService : AbstractSnippetInfoService
             IThreadingContext threadingContext,
             SVsServiceProvider serviceProvider,
             IAsynchronousOperationListenerProvider listenerProvider)
-            : base(threadingContext, serviceProvider, Guids.CSharpLanguageServiceId, listenerProvider)
+            : base(threadingContext, (IAsyncServiceProvider)serviceProvider, Guids.CSharpLanguageServiceId, listenerProvider)
```

Comment:
this appears to be used in other places, e.g. http://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/VisualStudioDecompilerEulaService.cs,31

So unless I hear otherwise I'll keep this